src/Sarif/HashUtilities.cs

```diff
@@ -31,15 +31,14 @@ internal static IFileSystem FileSystem
 
         public static IDictionary<string, HashData> MultithreadedComputeTargetFileHashes(IEnumerable<string> analysisTargets, bool suppressConsoleOutput = false)
         {
-            if (analysisTargets == null) { return null; }
```

Comment:
>null [](http://example.com/codeflow?start=50&length=4)

Please revert this change. You should be careful about allocating objects to indicate no results. Why not just have the caller handle null? This code succeeded previously, so you probably have only just introduced the requirement for this to be non-null.

Reply:
reverted